apps/client/src/features/rundown/event-editor/EventEditor.tsx

```diff
@@ -87,24 +89,47 @@ export default function EventEditor(props: EventEditorProps) {
             </Button>
           )}
         </Editor.Title>
+
         {Object.keys(customFields).map((fieldKey) => {
           const key = `${event.id}-${fieldKey}`;
           const fieldName = `custom-${fieldKey}`;
           const initialValue = event.custom[fieldKey] ?? '';
           const { backgroundColor, color } = getAccessibleColour(customFields[fieldKey].colour);
           const labelText = customFields[fieldKey].label;
 
-          return (
-            <EventTextArea
-              key={key}
-              field={fieldName}
-              label={labelText}
-              initialValue={initialValue}
-              submitHandler={handleSubmit}
-              className={style.decorated}
-              style={{ '--decorator-bg': backgroundColor, '--decorator-color': color } as CSSProperties}
-            />
-          );
+          if (customFields[fieldKey].type === 'string') {
+            return (
+              <EventTextArea
+                key={key}
+                field={fieldName}
+                label={labelText}
+                initialValue={initialValue}
+                submitHandler={handleSubmit}
+                className={style.decorated}
+                style={{ '--decorator-bg': backgroundColor, '--decorator-color': color } as CSSProperties}
+              />
+            );
+          }
+
+          if (customFields[fieldKey].type === 'image') {
+            return (
+              <div key={key} className={style.customImage}>
+                <EventTextInput
```

Comment:
The length limit on the text input seams a bit to strict considering image url's